examples/defaulter-gen/generators/defaulter.go

```diff
@@ -497,6 +498,23 @@ func mustEnforceDefault(t *types.Type, depth int, omitEmpty bool) (interface{},
 	}
 }
 
+var refRE = regexp.MustCompile(`^ref\((?P<reference>[^"]+)\)$`)
+var refREIdentIndex = refRE.SubexpIndex("reference")
+
+// parseAsRef looks for strings that match one of the following:
+//   - ref(Ident)
+//   - ref("pkgpath".Ident)
```

Comment:
comment is now out of date?
```suggestion
//   - ref(pkgpath.Ident)
```